tests/test_traced_data.py

```diff
@@ -6,6 +6,12 @@
 from core_data_modules.traced_data import TracedData, Metadata
 
 
+class TestMetadata(unittest.TestCase):
+    def test_location(self):
+        location = Metadata.location()
+        self.assertTrue(location.endswith("tests/test_traced_data.py:11:test_location"))
```

Comment:
That looks like a relative path rather than an absolute one? I'm fine with that, but we probably shouldn't call it an absolute path

Reply:
It is absolute, but the test only checks "endswith" in order to be independent of where the project lives.

I'll add a comment to the code.